### PR TITLE
Create & set up a clinic database

### PR DESCRIPTION
**Pull request by ouasamine**
## Overview
In this part, we had to create a database for a clinic base on a given diagram.
## Workflow

- Created the base tables from the given diagram
- Introduced a join table to manage a many-to-many relation between medical_histories and treatments

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @ouasamine & @arabiu033, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both partners. Kindly make sure both contribute as it's a pair programming project. Also, the git-flow is not correct it should be from the feature branch to the dev not the main.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by arabiu033** [comment]
> ## Invalid Code Review Request 
> Hi @ouasamine & @arabiu033, This is @sja-thedude your code reviewer.
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> You have submitted a project, that doesn't have commits from both partners. Kindly make sure both contribute as it's a pair programming project. Also, the git-flow is not correct it should be from the feature branch to the dev not the main.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Hi @sja-thedude ,
Because the task is brief, we decided to have only one driver. But I've now made some contributions. Have a great Day :+1:

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  treatment_id INT REFERENCES treatments(id),
+  PRIMARY KEY (id)
+);
+
+-- treatment table
+CREATE TABLE treatments (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  type VARCHAR(100),
+  name VARCHAR(100),
+  PRIMARY KEY (id)
```
Comment (youmari) [inline schema_based_on_diagram.sql]: - [ ] it would be better if you add `indexes` to the all foreign keys so that we can improve performance

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @ouasamine , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**